samples/client/petstore/java/jersey1/src/test/java/io/swagger/client/ApiClientTest.java

```diff
@@ -203,25 +246,12 @@ public void testParameterToPairsWhenValueIsEmptyStrings() throws Exception {
     }
 
     @Test
-    public void testParameterToPairsWhenValueIsNotCollection() throws Exception {
-        String name = "param-a";
-        Integer value = 1;
-
-        List<Pair> pairs = apiClient.parameterToPairs("csv", name, value);
-
-        assertEquals(1, pairs.size());
-        assertEquals(value, Integer.valueOf(pairs.get(0).getValue()));
-    }
-
-    @Test
     public void testParameterToPairsWhenValueIsCollection() throws Exception {
         Map<String, String> collectionFormatMap = new HashMap<String, String>();
         collectionFormatMap.put("csv", ",");
         collectionFormatMap.put("tsv", "\t");
         collectionFormatMap.put("ssv", " ");
-        collectionFormatMap.put("pipes", "\\|");
-        collectionFormatMap.put("", ","); // no format, must default to csv
-        collectionFormatMap.put("unknown", ","); // all other formats, must default to csv
```

Comment:
@wing328 I think parameterToPairs still technically defaults to csv in those cases. If you want I can add the tests back.

However, I think this behaviour is wrong. I believe it is impossible to get that far without a defined collection format; if it's not defined DefaultCodegen appears to define it as "csv" for you. As such I don't think we need to handle the empty string case. Moreover, the invalid case should throw an error -- we shouldn't just silently assume csv. Don't we validate the spec before generating APIs from it first?